cores/arduino/stm32/usb/cdc/usbd_cdc_if.c

```diff
@@ -245,6 +249,17 @@ void CDC_deInit(void)
   }
 }
 
+bool CDC_connected()
+{
+  uint32_t transmitTime = 0;
+  if (transmitStart) {
+    transmitTime = HAL_GetTick() - transmitStart;
+  }
+  return hUSBD_Device_CDC.dev_state == USBD_STATE_CONFIGURED
+         && transmitTime < USB_CDC_TRANSMIT_TIMEOUT

```

Comment:
USB_CDC_TRANSMIT_TIMEOUT is 3.
Is it not too low on slower board ?

Maybe ti could be fine to be able to redefine it if needed
```suggestion
#ifndef
#define USB_CDC_TRANSMIT_TIMEOUT 3 
#endif
```

Reply:
The value depends on how fast/slow the host reads from the CDC endpoint. I added some comments and the #ifndef suggestion.